src/ClusterBootstrap/deploy.py

```diff
@@ -284,82 +284,6 @@ def create_cluster_id():
         config["clusterId"] = utils.get_cluster_ID_from_file()
         print "Cluster ID is " + config["clusterId"]
 
-def add_acs_config(command):
```

Comment:
Are we sure that this method is no longer needed?

Reply:
yes, no longer needed. not only this one, but also others like hdfs glusterdfs etc.